airflow/www/views.py

```diff
@@ -5827,6 +5828,33 @@ def add_user_permissions_to_dag(sender, template, context, **extra):
     context["dag"] = dag
 
 
+##############################################################################
+#                                                                            #
+#                          Development Views                                 #
+#                                                                            #
+##############################################################################
+
+
+class DevView(BaseView):
+    """View to show Airflow Dev Endpoints.
+
+    This view should only be accessible in development mode. For
```

Comment:
Is there anything to actually enforce this? I don’t see one.

Reply:
There's no enforcement yet. It seems there's no concept of dev mode in airflow. I tried allowing access to localhost only using request.remote_addr but that didn't work

Reply:
One simple solution would be to add a config.

Reply:
I decided to introduce `AIRFLOW_ENV` instead of adding a new config. When set to `development`, the view becomes accessible.